src/player.h

```diff
@@ -594,6 +594,7 @@ class Player final : public Creature, public Cylinder
 		bool isImmune(ConditionType_t type) const final;
 		bool hasShield() const;
 		bool isAttackable() const final;
+		bool lastHitIsPlayer(Creature* lastHitCreature) const;
```

Comment:
I see, well I decided to make it static instead to avoid any code inconsistency. :smile: